inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/SortClusterServiceImpl.java

```diff
@@ -249,17 +250,29 @@ private SortClusterConfig getConfigByClusterNameV2(
     private List<Map<String, String>> parseIdParamsV2(List<StreamSinkEntity> streams) {
         return streams.stream()
                 .map(streamSink -> {
-                    StreamSinkOperator operator = sinkOperatorFactory.getInstance(streamSink.getSinkType());
-                    List<String> fields = fieldMap.get(streamSink.getInlongGroupId());
-                    return operator.parse2IdParams(streamSink, fields);
+                    try {
+                        StreamSinkOperator operator = sinkOperatorFactory.getInstance(streamSink.getSinkType());
+                        List<String> fields = fieldMap.get(streamSink.getInlongGroupId());
+                        return operator.parse2IdParams(streamSink, fields);
+                    } catch (Exception e) {
+                        LOGGER.error("fail to parse id params of groupId={}, streamId={} name={}, type={}}",
+                                streamSink.getInlongGroupId(), streamSink.getInlongStreamId(),
+                                streamSink.getSinkName(), streamSink.getSinkType(), e);
+                        return new HashMap<String, String>();
```

Comment:
Why does it return normally when parsing parameters fails？

Reply:
Fixed, should print logs and filter out all dirty records when parsing them, instead of return empty objects.